Issue #46 fixed

edouardalvescamilo opened the pull request:
Added a click listener on "menu_top_left" in order to remove the class "scrolledleft" from "welcome".

AlexManga committed:
Issue #91

add class hidden to top menu and bottom menu

Yoric committed:
Added AlexManga to the list of contributors

indydedeken committed:
Issue #64 Favicon

add favicon

Yoric commented:
That's probably good, but I can't accept the patch because of 54e9658 and 0de2a3a. Can you remove or squish these commits before proceeding?

Commit:
Issue #39 resolved

Yoric commented:
I still see issue #107 in the pull request, which is surprising.

Yoric committed:
Added Axel Ajavon, Karim Smaili, Maxime Lecoq to the list of authors

Yoric committed:
Added Antoine Mazelin to the list of authors

Yoric commented on app.js:
```diff
@@ -202,6 +202,10 @@ $("menu_top_right").addEventListener("click", event => {
 });
 
+$("menu_top_left").addEventListener("click", event => {
+	$("welcome").classList.remove("scrolledleft");
```
Could you fix indentation?

edouardalvescamilo committed:
Issue #46

Added a mouse click listener on "menu_top_left" in order to remove the
class "scrolledleft" from "welcome"

edouardalvescamilo committed:
Issue #46

Added a mouse click listener on "menu_top_left" in order to remove the
class "scrolledleft" from "welcome"

Yoric commented:
Could you please rebase again?

Yoric commented:
There are too many changes in this commit. Can you rebase it?

edouardalvescamilo committed:
Merge branch 'gh-pages' of https://github.com/edouardalvescamilo/lector-dev into gh-pages

Yoric commented:
@AlexManga I'm still waiting for you to squash your changes.

AlexManga commented:
@edouardalvescamilo the last comment is for you.
